Firmware/RTK_Surveyor/System.ino

```diff
@@ -293,18 +293,35 @@ bool isConnected(uint8_t deviceAddress)
 //Create a test file in file structure to make sure we can
 bool createTestFile()
 {
-  SdFile testFile;
-  char testFileName[40] = "testfile.txt";
-
-  //Attempt to write to the file system
-  if (testFile.open(testFileName, O_CREAT | O_APPEND | O_WRITE) != true)
-    return (false);
-
-  //File successfully created
-  testFile.close();
-  if (sd->exists(testFileName))
-    sd->remove(testFileName);
-  return (true);
+  if (USE_SPI_MICROSD)
+  {
+    SdFile testFile;
+    char testFileName[40] = "testfile.txt";
+  
+    //Attempt to write to the file system
+    if (testFile.open(testFileName, O_CREAT | O_APPEND | O_WRITE) != true)
+      return (false);
+  
+    //File successfully created
+    testFile.close();
+    if (sd->exists(testFileName))
+      sd->remove(testFileName);
+    return (!sd->exists(testFileName));
+  }
+  else
+  {
+    // SDIO MMC
+    char testFileName[40] = "/testfile.txt"; // Create it in the root directory
```

Comment:
I believe sdFat handles /fileName.ext just fine. You could consolidate some of the char[]s.

Reply:
OK. I'll test this once I have functional Reference_Station code.
My plan at the moment is to have duplicate names for SdFat vs SD_MMC, without and with the slash. But if SdFat does ignore the slash, then we can use common names.
I'm adding this to the [sheet](https://docs.google.com/spreadsheets/d/1TNjSrNvyVTpTsuWSNNbTeBClg1CfebEc8Dg3rV33QRg/edit#gid=447334257) and will track it there.